src/main/fc/fc_rc.c

```diff
@@ -517,4 +530,28 @@ void initRcProcessing(void)
 
         break;
     }
+
+    interpolationChannels = 0;
+    switch (rxConfig()->rcInterpolationChannels) {
+    case INTERPOLATION_CHANNELS_RPYT:
+        interpolationChannels |= THROTTLE_FLAG;
+
+        FALLTHROUGH;
+    case INTERPOLATION_CHANNELS_RPY:
+        interpolationChannels |= YAW_FLAG;
+
+        FALLTHROUGH;
+    case INTERPOLATION_CHANNELS_RP:
+        interpolationChannels |= ROLL_FLAG | PITCH_FLAG;
+
+        break;
+    case INTERPOLATION_CHANNELS_T:
+        interpolationChannels |= THROTTLE_FLAG;
+
+        FALLTHROUGH;
+    case INTERPOLATION_CHANNELS_RPT:
+        interpolationChannels |= ROLL_FLAG | PITCH_FLAG;
+
+        break;
+    }
```

Comment:
This switch logic isn't working correctly.  For `INTERPOLATION_CHANNELS_T` the `interpolationChannels` variable is getting set to 11 meaning RPT.  For `INTERPOLATION_CHANNELS_RPT` it's getting set to 3 for RP.

The other cases work as expected.

Reply:
This part is working now.